source/CMakeLists.txt

```diff
@@ -39,4 +39,9 @@ target_link_libraries(binder
   clangTooling
   clangBasic
   clangASTMatchers
+  clangAST
+  clangFrontend
```

Comment:
Why `clangAST` and `clangFrontend` was added here? They are not needed for Binder build.

Reply:
Basically the linker advices to do so:

/usr/bin/ld: CMakeFiles/binder.dir/class.cpp.o: undefined reference to symbol '_ZNK5clang11DeclContext18isDependentContextEv'
/opt/rh/llvm-toolset-7/root/usr/lib64/libclangAST.so.5: error adding symbols: DSO missing from command line

/usr/bin/ld: CMakeFiles/binder.dir/binder.cpp.o: undefined reference to symbol '_ZN5clang22PCHContainerOperationsC1Ev'
/opt/rh/llvm-toolset-7/root/usr/lib64/libclangFrontend.so.5: error adding symbols: DSO missing from command line

It looks like it is related to the way the clang is compiled.